usr/local/www/reboot.php

```diff
@@ -49,24 +52,33 @@
 $pgtitle = array(gettext("Diagnostics"),gettext("Reboot System"));
 include("head.inc");
 
+if (stristr($_POST['Submit'], gettext("Yes"))) {
+	?><meta http-equiv=\"refresh\" content=\"70;url=/\"> <?php
+	print('<div class="alert alert-success" role="alert">'.gettext("The system is rebooting now. This may take one minute or so.").'</div>');
+
+	if(DEBUG)
+	   print("Not actually rebooting (DEBUG is set true)<br>");
+	else
+		system_reboot();
+} else {
+
+
 ?>
-<body link="#0000CC" vlink="#0000CC" alink="#0000CC">
-<?php include("fbegin.inc"); ?>
-<?php if ($_POST['Submit'] == " " . gettext("Yes") . " "): ?>
-<meta http-equiv=\"refresh\" content=\"70;url=/\">
-<?php	print_info_box(gettext("The system is rebooting now. This may take one minute.")); ?>
-<pre>
-<?php 	system_reboot(); ?>
-</pre>
-<?php else: ?>
-<form action="reboot.php" method="post">
-	<p><strong><?=gettext("Are you sure you want to reboot the system?");?></strong></p>
-	<p>
-	<input name="Submit" type="submit" class="formbtn" value=" <?=gettext("Yes");?> " />
-	<input name="Submit" type="submit" class="formbtn" value=" <?=gettext("No");?> " />
-	</p>
-</form>
-<?php endif; ?>
-<?php include("fend.inc"); ?>
-</body>
-</html>
+
+<div class="panel panel-default">
+	<div class="panel-heading">Are you sure you want to reboot the system?</div>
+	    <div class="panel-body">Click "Yes" to reboot the system immediately, or "No" to go to the system dashboard without rebooting. (There will be a brief delay before the dashboard appears.)<br /><br />
+            <form action="reboot.php" method="post">
+                <input type="submit" class="btn btn-danger pull-center" name="Submit" value="Yes">
+                <input type="submit" class="btn btn-default" name="Submit" value="No">
```

Comment:
Instead of a button that does nothing I would remove that; or transform it to ```<a href="/" class="btn btn-defaul">```. This would mean you can rewrote line 55 to simply ```if ($_SERVER['REQUEST_METHOD'] == 'POST')```